docs/source/index.rst

```diff
@@ -21,17 +21,25 @@ Flow is a work in progress - input is welcome. Available documentation is limite
    flow_setup
    Tutorials <https://github.com/flow-project/flow/tree/master/tutorials/>
    multiagent
+   rendering
    visualizing
    modules
 
 
 .. Citing Flow
 .. ============
 
```

Comment:
Can we uncomment this so it actually shows up?

Reply:
It seems this will be fixed after merging. If not, will create an issue.